tests/unit/models/summon.js

```diff
@@ -1960,6 +2138,133 @@ describe("Summon Model >", function() {
     });
   });
 
+  describe("summon model eocLinkTemplate method >", function() {
+    beforeEach(function() {
+      delete browzine.articleExpressionOfConcernEnabled;
+      delete browzine.articleExpressionOfConcernWording;
+      delete browzine.articleExpressionOfConcernText;
+      delete browzine.version;
+    });
+
+    afterEach(function() {
+      delete browzine.articleExpressionOfConcernEnabled;
+      delete browzine.articleExpressionOfConcernWording;
+      delete browzine.articleExpressionOfConcernText;
+      delete browzine.version;
+    });
+
+    it("should build an eoc link template for article search results", function() {
+      var scope = {
+        document: {
+          content_type: "Journal Article",
+          dois: ["10.1002/ijc.25451"]
+        }
+      };
+
+      var data = summon.getData(eocArticleResponse);
+      var articleEOCNoticeUrl = summon.getArticleEOCNoticeUrl(scope, data);
+      var template = summon.eocLinkTemplate(articleEOCNoticeUrl);
+
+      var $template = document.createElement("div");
+      $template.innerHTML = template;
+
+      expect(data).toBeDefined();
+      expect(articleEOCNoticeUrl).toBeDefined();
+
+      expect(template).toBeDefined();
+
+      expect(template).toContain("https://develop.libkey.io/libraries/XXXX/10.1002/ijc.25451");
+
+      expect($template.innerText).toContain("Expression of Concern More Info");
+    });
+
+    it("should apply the articleExpressionOfConcernWording config property", function() {
+      browzine.version = 2;
```

Comment:
Related to the bit up top, I think we can get rid of this browzine.version bit for the new config properties.